### PR TITLE
Update mockResponse to use passed in args

### PR DESCRIPTION
**Pull request by saivickna**

### CODE REVIEW

File: src/routes/graphiql.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ function makeGraphiqlOptions(config) {
     return req => ({
         /* Connect GraphiQL to GraphQL.
          */
-        endpointURL: config.routes.graphiql.endpointURL,
+        endpoint: config.routes.graphiql.endpoint,
```
Comment (jessemyers) [inline src/routes/graphiql.js]: I don't think this is right. The left-hand-side presumably still needs the `URL`

File: src/testing/openapi.js [hunk 2]
```diff
@@ -8,8 +8,8 @@ import { OpenAPIError } from '@globality/nodule-openapi';
 export function mockResponse(name, operationId, data) {
     const obj = {};
 
-    set(obj, `clients.mock.${name}.${operationId}`, jest.fn(async (req, args) => ({
-        data: isFunction(data) ? data(args) : data,
+    set(obj, `clients.mock.${name}.${operationId}`, jest.fn(async req => ({
+        data: isFunction(data) ? data(req.params) : data,
```
Comment (jessemyers) [inline src/testing/openapi.js]: Why is it safe to assume that `req.params` exists? Normally we try to pull `args` from the usual resolver arguments.

File: src/routes/graphiql.js (hunk 1 above)
Comment (cdm-ium) [reply src/routes/graphiql.js]: Agreed

Comment (saivickna) [reply src/routes/graphiql.js]: sorry, this wasnt supposed to be included in this

File: src/testing/openapi.js (hunk 2 above)
Comment (saivickna) [reply src/testing/openapi.js]: @GabeLambda and I were debugging this, and for some reason this is where it seems to attach the parameters to, and args is undefined. We can safely assume that `req` exists, so even if we don't pass in any parameters and `params` is undefined, JS will not fail, will just be `undefined`.

Comment (jessemyers) [reply src/testing/openapi.js]: That makes me think there's a bug somewhere else.

File: src/clients/__tests__/openapi.test.js [hunk 3]
```diff
@@ -62,6 +62,21 @@ describe('createOpenAPIClient', () => {
         });
     });
 
+    it('supports mocking a post response with a function', async () => {
+        const config = await Nodule.testing().fromObject(
+            mockResponse('petstore', 'pet.create', body => ({ items: [JSON.parse(body).name] })),
```
Comment (saivickna) [inline src/clients/__tests__/openapi.test.js]: @jessemyers Should I always parse the body and move this into `src/testing/openapi.js`?

Comment (jessemyers) [reply src/clients/__tests__/openapi.test.js]: That sounds useful, yes.

**Pull request merged**